allalarm.spec

```diff
@@ -282,7 +282,7 @@
         "create": false,
         "delete": false,
         "description": "The alarm API allows the management of system alarms.",
-        "entity_name": "Alarm",
+        "entity_name": "AllAlarm",
```

Comment:
I have seen this as well. Also in the API itself, there is the 'allalarm' url path available, and I think the alarm one is as well... This has to be reflected correctly in these specs.

Regards,
Philippe

Sent from a mobile device

> On 09 Sep 2016, at 23:44, martin-leclerc <notifications@github.com> wrote:
> 
> In allalarm.spec:
> 
> > @@ -282,7 +282,7 @@
> >          "create": false,
> >          "delete": false,
> >          "description": "The alarm API allows the management of system alarms.",
> > -        "entity_name": "Alarm",
> > +        "entity_name": "AllAlarm",
> @natabal @little-dude The reason the entity name was changed from “Alarm” to “AllAlarm” is because it conflicts with another entity called “Alarm”. Having two entities with the same name causes problems in the generated code of the VSPKs. Could we simply removed one of them? From: natabal [mailto:notifications@github.com] Sent: September 9, 2016 4:43 PM To: nuagenetworks/vsd-api-specifications Cc: martin-leclerc; Mention Subject: Re: [nuagenetworks/vsd-api-specifications] Fix CI and fix specs for Java VSPK generation (#10) In allalarm.spec <#10 (comment)> :
> @@ -282,7 +282,7 @@ "create": false, "delete": false, "description": "The alarm API allows the management of system alarms.", - "entity_name": "Alarm", + "entity_name": "AllAlarm",
> yes! The entity name is 'Alarm' — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <https://github.com/nuagenetworks/vsd-api-specifications/pull/10/files/ecb1e2c511c73aaf3eacade98c4c3f5c58f82665#r78245538> , or mute the thread <https://github.com/notifications/unsubscribe-auth/APGZ0qWVaNuXLnE-ZNCRE7x1lrj2ff_Eks5qocS7gaJpZM4Jz_dv> . <https://github.com/notifications/beacon/APGZ0rf-jm0GfZPs-2rVgrEyI7ET1wkyks5qocS7gaJpZM4Jz_dv.gif>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>